roles/common/tasks/configure.yml

```diff
@@ -285,13 +285,14 @@
 
```

Comment:
Also, since we are removing the logic to add `rock_mgmt_nets` as a source to the work zone, we need to add that back as a separate task:

```
- name: Configure sources for work zone
  firewalld:
    source: "{{ rock_mgmt_nets }}"
    zone: work
    state: enabled
```